explorerlib/index.go

```diff
@@ -45,9 +46,12 @@ func (index *Index) FetchResponse(queryExpression string, numberOfResults int) (
 
 func (index *Index) BuildGoodQueries(wordCountsByLanguage map[string]WordCounts, numberOfQueryByLanguage int, averageNumberOfWords int) (map[string][]string, error) {
 	queriesInLanguage := make(map[string][]string)
+
+	tic := 0
```

Comment:
Before the for loop, add a logging message: "Building queries and calling the index to validate that they return results"
Calculate the total number of calls that will be done to the index and log it.